Rakefile

```diff
@@ -9,18 +9,36 @@ require "packaging/configuration"
 Packaging::Tasks.load_tasks(:exclude => ["tarball.rake"])
 
 require "yast/tasks"
-Yast::Tasks.submit_to(ENV.fetch("YAST_SUBMIT", "factory").to_sym)
+yast_submit = ENV.fetch("YAST_SUBMIT", "factory").to_sym
+Yast::Tasks.submit_to(yast_submit)
 
 Packaging.configuration do |conf|
   conf.package_name.sub!(/-.*/, "") # strip branch name
   conf.package_dir    = ".obsdir" # Makefile.ci puts it there
   conf.skip_license_check << /.*/
 
   # defined in Rakefile in https://github.com/openSUSE/packaging_rake_tasks
-  conf.obs_api = "https://api.opensuse.org/"
-  conf.obs_project = "filesystems:snapper"
-  conf.obs_target = "openSUSE_Factory"
-  conf.obs_sr_project = "filesystems:snapper"
+  if yast_submit == :factory
+    # OBS: filesystems:snapper/snapper
+    conf.obs_api = "https://api.opensuse.org/"
+    conf.obs_project = "filesystems:snapper"
+    conf.obs_sr_project = "filesystems:snapper"
+    conf.obs_target = "openSUSE_Factory"
+  else
+    # Using values from
+    # https://github.com/yast/yast-rake/blob/master/data/targets.yml
+    # loaded by Yast::Tasks.submit_to()
+  end
+end
+
+desc 'Show configuration'
+task :show_config do
+  Packaging.configuration do |conf|
+    print "API: #{conf.obs_api}\n"
```

Comment:
Yes, right; in a previous version, I had a headline with more complex formatting, but that one is gone now.